### PR TITLE
Cleaning up more deprecation warnings

### PR DESCRIPTION
**Pull request by andrewlee94**
## Proposed changes:
- Update a lot of Jupyter notebooks with new APIs.

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: src/Examples/UnitModels/Reactors/cstr_testing.ipynb [hunk 1]
```diff
@@ -83,9 +83,8 @@
     "from pyomo.network import Arc\n",
     "\n",
     "from idaes.core import FlowsheetBlock\n",
-    "from idaes.models.properties.modular_properties.base.generic_property import (\n",
-    "        GenericParameterBlock)\n",
-    "from idaes.models.properties.modular_properties.base.generic_reaction import (\n",
+    "from idaes.models.properties.modular_properties import (\n",
```
Comment (bpaul4) [inline src/Examples/UnitModels/Reactors/cstr_testing.ipynb:86]: It looks like updating (in somes cases, reverting) these imports to `from idaes.models.properties.modular_properties.base.generic_property import GenericParameterBlock` will resolve the visible issues in the checks.

Comment (andrewlee94) [reply src/Examples/UnitModels/Reactors/cstr_testing.ipynb:86]: See https://github.com/IDAES/idaes-pse/pull/942

Comment (bpaul4) [reply src/Examples/UnitModels/Reactors/cstr_testing.ipynb:86]: Ah, I see. That should resolve it then, thanks for the link.

**Review by lbianchi-lbl — APPROVED** [review]
From a cursory look, everything looks good.

**Review by bpaul4 — APPROVED** [review]
Everything looks good.

**Pull request merged**